src/ui/public/vislib/styles/_legend.less

```diff
@@ -6,9 +6,17 @@ visualize-legend {
   flex-direction: row;
 }
 
+.legend-title {
+  display: flex;
+  flex-direction: row;
+
+  i {
```

Comment:
icons are one of the slower css selectors, is there a class or something else you could use?